### DISCUSSION TITLE
Nichts Funktioniert

### DISCUSSION THREAD

**Discussion by bysja23** [flybywiresim/aircraft#6261] [Q&A]
Hello I ask for your help.
The problem is that the plane doesn't have any functions for me. The display is all dark, the battery is at 0V. Even if I start the flight in the air, the aircraft flies with the systems switched off.
Does anyone have a suggestion for me?

**Comment by andrejetcat** [comment]
i would suggest to completly delete the "flybywire-aircraft-a320-neo" - folder from your community folder and reinstall the Stable or the Development version again via the FBW installer. 
Give it a try

**Comment by bysja23** [comment]
I did that, but it didn't work
